CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.7.1 (2018-05-18)
```

Comment:
Can you change it to `2.8.0`? we're adding new functionality, not fixing a bug :)

Reply:
Sure @diegoserranoa thanks for pointing it out.